core/src/main/java/tc/oc/pgm/kits/Slot.java

```diff
@@ -60,10 +67,9 @@ public boolean isAuto() {
     return index < 0;
   }
 
-  // If we support more inventory types, we would probably widen the type accepted by this method
```

Comment:
Widen the type does not mean make the whole class generic

Simply change `HumanEntity` to `InventoryHolder` and call it a day, having to change the whole codebase to make Slot a generic class is unneeded

Reply:
The idea is that the different Slot subclasses can then require their own subclass of InventoryHolder, it would be weird for the Player class to accept InventoryHolder for example.

Reply:
I get the reason, but it's simply too much overhead (in terms of complexity) that provides little benefit